ql/pricingengines/vanilla/fdhestonmixedvanillaengine.hpp

```diff
@@ -45,28 +45,31 @@ namespace QuantLib {
     */
     class FdmQuantoHelper;
 
-    class FdHestonVanillaEngine
-        : public GenericModelEngine<HestonModel,
+    class FdHestonMixedVanillaEngine : public GenericModelEngine<HestonModel,
                                     DividendVanillaOption::arguments,
                                     DividendVanillaOption::results> {
       public:
         // Constructor
-        explicit FdHestonVanillaEngine(
+        explicit FdHestonMixedVanillaEngine(
```

Comment:
I would keep the original name for backwards compatibility. (With this change, existing valid programs using the library might no longer compile. We try to avoid that.)  It can still be called a Heston model, even if generalized, I think?  The same goes for the file name.

Reply:
Reverted.